pkg/plugins/policies/meshaccesslog/api/v1alpha1/validator.go

```diff
@@ -153,6 +160,8 @@ func validateFormat(format Format) validators.ValidationError {
 				verr.AddViolationAt(path, `is not a valid JSON object`)
 			}
 		}
+	default:
+		verr.AddViolationAt(validators.Root(), fmt.Sprintf("unknown backend type %v", format.Type))
```

Comment:
isn't this made impossible by k8s/our schema validator on universal?

Reply:
Yes, but we don't call schema validation from `validator_test.go`, so this line helps a bit to write testcases

Reply:
but then it can panic instead IMO